src/routes/userRoutes.js

```diff
@@ -14,5 +14,6 @@ userRouter.get('/view-profile', decode, userController.viewProfile);
 userRouter.post('/edit-profile-image', decode, multerUploads, userController.uploadProfileImage);
 userRouter.post('/create/facility', decode, userController.createFacility);
 userRouter.post('/create/facility/room', decode, userController.createRoom);
+userRouter.patch('/assign/manager', decode, userController.assignManager);
```

Comment:
hey @Ntare22 , can't you use the protectRoute middleware, it will help in removing code repetition

Reply:
Then here, we can just use the ```/setRoles``` route

Reply:
Then I'd suggest just ```/assign-manager``` since assign is not a collective.